src/dialect/hive.rs

```diff
@@ -51,4 +51,8 @@ impl Dialect for HiveDialect {
     fn require_interval_qualifier(&self) -> bool {
         true
     }
+
+    fn supports_bang_not_operator(&self) -> bool {
```

Comment:
Do we have a link to the syntax? If so it would be good to include that here as part of the documentation

Reply:
Fixed!